scripts/bb-test-bootable-image.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Check for a local cached configuration.
+if test -f /etc/buildslave; then
+    . /etc/buildslave
+fi
+
+# Custom test options will be saved in the tests directory.
+if test -f "../TEST"; then
+    . ../TEST
+fi
+
+TEST_BOOTABLE_IMAGE_SKIP=${TEST_BOOTABLE_IMAGE_SKIP:-"No"}
+if echo "$TEST_BOOTABLE_IMAGE_SKIP" | grep -Eiq "^yes$|^on$|^true$|^1$"; then
+    echo "Skipping disabled test"
+    exit 3
+fi
+
+ZPOOL=${ZPOOL:-"zpool"}
+ZFS=${ZFS:-"zfs"}
+CONSOLE_LOG="$PWD/console.log"
+OUTPUT_LOG="$PWD/output.log"
+
+case "$BB_NAME" in
+Fedora*)
+    true
+    ;;
+
+*)
+    echo "$BB_NAME unknown platform, skipping image test" >>$OUTPUT_LOG 2>&1
+    exit 18
+    ;;
+
+# Cleanup the pool and restore any modified system state.  The console log
+# is dumped twice to maximize the odds of preserving debug information.
+cleanup()
+{
+    dmesg >$CONSOLE_LOG
+    sudo -E $ZPOOL destroy -f "$TEST_BOOTABLE_IMAGE_POOL" &>/dev/null
+    sudo -E rm -f /etc/zfs/zpool.cache "$TEST_BOOTABLE_IMAGE_VDEV"
+    dmesg >$CONSOLE_LOG
+}
+trap cleanup EXIT SIGTERM
+
+set -x
+
+TEST_BOOTABLE_IMAGE_POOL=${TEST_BOOTABLE_IMAGE_POOL:-"bootable_image"}
+TEST_BOOTABLE_IMAGE_VDEV=${TEST_BOOTABLE_IMAGE_VDEV:-"/var/tmp/bootable_image.img"}
+TEST_BOOTABLE_IMAGE_OPTIONS=${TEST_BOOTABLE_IMAGE_OPTIONS:-""}
+
+echo -n >$OUTPUT_LOG 2>&1
+
+rm -rf rpms/                            >>$OUTPUT_LOG 2>&1
+mkdir -p rpms/                          >>$OUTPUT_LOG 2>&1
+cp -Rv ../spl/*.rpm ../zfs/*.rpm rpms/  >>$OUTPUT_LOG 2>&1
```

Comment:
This should work if we update the [bb-build-packages.sh](https://github.com/zfsonlinux/zfs-buildbot/blob/master/scripts/bb-build-packages.sh#L50) script to keep the noarch dkms package it built.